lib/rules/constructor-super.js

```diff
@@ -274,15 +300,15 @@ module.exports = function(context) {
                         message: "Unexpected duplicate 'super()'.",
                         node: node
                     });
+                } else if (!funcInfo.superIsConstructor) {
+                    context.report({
+                        message: "Unexpected 'super()' because 'super' is not a constructor.",
```

Comment:
Hmm, I think the class and the constructor are different things in the mind of many people, so class is not a constructor always?

In fact, this message was copied from the message of Chrome.

![image](https://cloud.githubusercontent.com/assets/1937871/14227119/71dd63a0-f930-11e5-95d0-1b0a55288610.png)